.pre-commit-config.yaml

```diff
@@ -0,0 +1,56 @@
+exclude: Pipfile\.lock|migrations|\.idea
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
+    hooks:
+      - id: trailing-whitespace
+      - id: check-executables-have-shebangs
+      - id: check-json
+      - id: check-case-conflict
+      - id: check-toml
+      - id: check-merge-conflict
+      - id: check-xml
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: check-symlinks
+      - id: mixed-line-ending
+      - id: fix-encoding-pragma
+        args:
+          - --remove
+      - id: pretty-format-json
+        args:
+          - --autofix
+      - id: requirements-txt-fixer
+
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.9.1
+    hooks:
+      - id: isort
+        name: Run isort to sort imports
+        files: \.py$
+        exclude: ^build/.*$|^.tox/.*$|^venv/.*$
+        args:
+          - --lines-after-imports=2
+          - --multi-line=3
+          - --trailing-comma
+          - --force-grid-wrap=0
+          - --use-parentheses
+          - --ensure-newline-before-comments
+          - --line-length=88
+
+  - repo: https://gitlab.com/pycqa/flake8
+    rev: 3.9.2
+    hooks:
+        - id: flake8
+          args:
+            - --per-file-ignores=*test*:E501,*__init__*:F401,*wsgi.py:E402
+            - --max-line-length=88
+            - --ignore=E203,W503
```

Comment:
I think these args won't be necessary as the hook will automatically use the config from `.flake8`.

Reply:
I won't, unfortunately. `pre-commit` runs hooks inside a virtualenv that doesn't include the root of your project, so it doesn't know any of your configuration files 😢

Reply:
I tried this locally and it seemed to work fine. Are you sure about it not working?

Reply:
@sarayourfriend for the hook, is it possible to provide a config path instead of defining the rules twice in two different locations? If not I think just leaving a note here should be enough that mentions the rules also being in the other file.

Reply:
I guess I'm not sure 😅 I'd tried it in the past and it didn't work and I found some SO answers about it to confirm the issue, but if you've got it working I'll try it again!

Reply:
Ah it worked! I removed the duplicated config. Thanks for testing it out!